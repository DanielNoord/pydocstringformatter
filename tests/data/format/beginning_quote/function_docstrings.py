def func():
    """
    A docstring."""

    def inner_func():
        """
        A docstring."""


def func():
    """
    A multi-line
    docstring
    """

    def inner_func():
        """A multi-line
        docstring"""
