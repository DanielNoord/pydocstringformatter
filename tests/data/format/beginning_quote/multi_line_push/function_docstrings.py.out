def func():
    """A multi-line
    docstring.
    """

    def inner_func():
        """A multi-line
        docstring.
        """
